src/util/index.js

```diff
@@ -76,3 +76,11 @@ export const formatFloatToString = (number: number): string => {
 export const hasProp = (obj: any, prop: string): boolean => {
   return {}.hasOwnProperty.call(obj, prop)
 }
+
+export const isDateBiggerThanToday = (newDate: any): boolean => {
+  if (newDate) {
+    return (moment(newDate).isAfter()) ? true : false
```

Comment:
??? We have new code blow according to your suggestions.  
 const isAfterYear = moment(newDate).isAfter(today, 'year')
    const isAfterMonth = moment(newDate).isAfter(today, 'month')
    const isAfterDay = moment(newDate).isAfter(today, 'day')
     return isAfterDay || isAfterMonth || isAfterYear